src/renderers/dom/client/utils/createObjectWithProto.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
The name is different, but this is effectively the same as Object.create, so this just introducing the polyfill.  We want to use ES6 classes instead as per https://github.com/facebook/react/issues/4189#issuecomment-120479693